pages/linux/lxi.md

```diff
@@ -0,0 +1,24 @@
+# lxi
+
+> Tool for controling LXI compatible instruments such as oscilloscopes.
+> More information: <https://github.com/lxi-tools/lxi-tools>.
+
+- Discover LXI devices on available networks:
+
+`lxi discover`
+
+- Capture a screenshot using plugin autodetection:
+
+`lxi screenshot --address {{ip_adress}}`
+
+- Capture a screenshot using specifying a plugin:
+
+`lxi screenshot --address {{ip_adress}} --plugin {{rigol-1000z}}`
+
+- Send SCPI command to an instrument:
```

Comment:
```suggestion
- Send a SCPI command to an instrument:
```

Reply:
Hello @CleanMachine1, my pleasure.
Thank you for another set of reviews.

I am not sure about this one, however. Previously I was corrected by @bl-ue that it should be `an SCPI`, because S is spelled as `es`. 

More at https://github.com/tldr-pages/tldr/pull/5215#discussion_r567959276

Is that related?

Reply:
`an` is correct, my suggestion has been updated, good spot rather than just accepting